analyzer/src/funTest/kotlin/GitRepoTest.kt

```diff
@@ -44,7 +45,9 @@ class GitRepoTest : StringSpec() {
     private lateinit var outputDir: File
 
     override fun beforeSpec(spec: Spec) {
-        outputDir = createTempDir()
+        // Do not use the class name as a suffix here to shorten the path. Otherwise the path will get too long for
+        // Windows to handle.
+        outputDir = createTempDir(ORT_NAME)
```

Comment:
I am surprised, why is the path too long here, but not for e.g. `NodeSupportTest` below?

Reply:
Because of the Git-internal submodule directory structure that gets created, not because of the committed project files themselves.

Reply:
I see, so it's actually a close call?

Reply:
Indeed...